.coderabbit.yaml

```
@@ -0,0 +1,27 @@
+    drafts: false
+  function_name_review:
+    enabled: false
+    criteria:
+      - "clarity"
+      - "consistency"
+      - "conventions"
+      - "verbs_only"
+  path_instructions:
+    - path: "**/*.java"
+      instructions: |
+        Review the the function name in the class and function names should be verb.    
```

yamllint

[error] trailing spaces

(trailing-spaces)